backend/app/app.go

```
@@ -0,0 +1,61 @@
+	signup := usecase.NewSignUp(pool, tokenConfig)
+
+	usecases := []UseCase{signup, signin}
+
+	return &App{pool, router, tokenConfig, usecases}
+}
+
+func (app *App) Run() {
+	app.SetupRoutes()
+
+	defer app.pool.Close()
+	app.router.Run()
```

golangci-lint

Error return value of `app.router.Run` is not checked

(errcheck)